cmd/bmx/bmx_default.go

```diff
@@ -3,20 +3,11 @@
 package main
 
 import (
-	"log"
 
 	"github.com/rtkwlf/bmx/config"
 	"github.com/rtkwlf/bmx/console"
 )
 
 func selectConsoleReader(userConfig config.UserConfig, limited bool) console.ConsoleReader {
-	if !limited {
-		return console.NewConsoleReader(false)
-	}
-
-	if !console.IsTtyAvailable() {
-		log.Fatal("Cannot create tty connection for writing output")
-	}
-
-	return console.NewConsoleReader(true)
+	return console.NewConsoleReader(limited)
```

Comment:
This should just impact the non-Mac cases, as the [darwin](https://github.com/rtkwlf/bmx/blob/940b43b847f7b2425db66824d04c6bf85e1e7df2/cmd/bmx/bmx_darwin.go#L10) logic handles the applescript selection.